<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>888</width>
    <height>530</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">color: rgb(200, 200, 200);
background-color: rgb(0, 64, 93);
</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="2" column="2">
     <widget class="QPushButton" name="alphabetUpDownSortButton">
      <property name="styleSheet">
       <string notr="true">color: rgb(200, 200, 200);
background-color: rgb(33, 33, 33);
font: 8pt &quot;Century Gothic&quot;;</string>
      </property>
      <property name="text">
       <string>Сорт. Я-А</string>
      </property>
     </widget>
    </item>
    <item row="2" column="4">
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="0" column="1">
       <widget class="QLineEdit" name="subjectLineEdit">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(200, 200, 200);
background-color: rgb(33, 33, 33);
font: 8pt &quot;Century Gothic&quot;;</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="0" column="0" rowspan="6">
     <layout class="QVBoxLayout" name="verticalLayout"/>
    </item>
    <item row="1" column="2">
     <widget class="QPushButton" name="alphabetDownUpSortButton">
      <property name="styleSheet">
       <string notr="true">color: rgb(200, 200, 200);
background-color: rgb(33, 33, 33);
font: 8pt &quot;Century Gothic&quot;;</string>
      </property>
      <property name="text">
       <string>Сорт. А-Я</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1" rowspan="2" colspan="2">
     <widget class="QListWidget" name="messageList">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="minimumSize">
       <size>
        <width>310</width>
        <height>0</height>
       </size>
      </property>
      <property name="sizeIncrement">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
      <property name="baseSize">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
      <property name="mouseTracking">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(200, 200, 200);
background-color: rgb(0, 36, 53);
font: 8pt &quot;Century Gothic&quot;;</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QPushButton" name="timeUpDownSortButton">
      <property name="styleSheet">
       <string notr="true">color: rgb(200, 200, 200);
background-color: rgb(33, 33, 33);
font: 8pt &quot;Century Gothic&quot;;</string>
      </property>
      <property name="text">
       <string>Сначала новые</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QPushButton" name="timeDownUpSortButton">
      <property name="styleSheet">
       <string notr="true">color: rgb(200, 200, 200);
background-color: rgb(33, 33, 33);
font: 8pt &quot;Century Gothic&quot;;</string>
      </property>
      <property name="text">
       <string>Сначала старые</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1" colspan="2">
     <widget class="QLabel" name="label_3">
      <property name="styleSheet">
       <string notr="true">color: rgb(200, 200, 200);
font: 8pt &quot;Century Gothic&quot;;</string>
      </property>
      <property name="text">
       <string>Сообщения</string>
      </property>
     </widget>
    </item>
    <item row="1" column="4">
     <layout class="QGridLayout" name="gridLayout_6">
      <item row="0" column="4" colspan="3">
       <widget class="QComboBox" name="comboBox">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(200, 200, 200);
background-color: rgb(33, 33, 33);
font: 8pt &quot;Century Gothic&quot;;</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="1" column="3">
     <widget class="QLabel" name="label_2">
      <property name="styleSheet">
       <string notr="true">color: rgb(200, 200, 200);
font: 8pt &quot;Century Gothic&quot;;</string>
      </property>
      <property name="text">
       <string>Получатель</string>
      </property>
     </widget>
    </item>
    <item row="2" column="3">
     <widget class="QLabel" name="label">
      <property name="styleSheet">
       <string notr="true">color: rgb(200, 200, 200);
font: 8pt &quot;Century Gothic&quot;;</string>
      </property>
      <property name="text">
       <string>Тема</string>
      </property>
     </widget>
    </item>
    <item row="0" column="4">
     <layout class="QGridLayout" name="gridLayout_7">
      <item row="1" column="2">
       <widget class="QPushButton" name="logoutButton">
        <property name="styleSheet">
         <string notr="true">color: rgb(200, 200, 200);
font: 8pt &quot;Century Gothic&quot;;
background-color: rgb(33, 33, 33);
</string>
        </property>
        <property name="text">
         <string>Выйти</string>
        </property>
       </widget>
      </item>
      <item row="1" column="3">
       <widget class="QPushButton" name="loginButton">
        <property name="styleSheet">
         <string notr="true">color: rgb(200, 200, 200);
background-color: rgb(33, 33, 33);
font: 8pt &quot;Century Gothic&quot;;</string>
        </property>
        <property name="text">
         <string>Войти</string>
        </property>
       </widget>
      </item>
      <item row="1" column="4">
       <widget class="QPushButton" name="registrationButton">
        <property name="styleSheet">
         <string notr="true">color: rgb(200, 200, 200);
background-color: rgb(33, 33, 33);
font: 8pt &quot;Century Gothic&quot;;</string>
        </property>
        <property name="text">
         <string>Регистрация</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLabel" name="usernameLabel">
        <property name="styleSheet">
         <string notr="true">color: rgb(200, 200, 200);
background-color: rgb(33, 33, 33);</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item row="4" column="3" colspan="2">
     <widget class="QTextEdit" name="messageTextArea">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(185, 185, 185);
background-color: rgb(0, 36, 53);
font: 8pt &quot;Century Gothic&quot;;</string>
      </property>
     </widget>
    </item>
    <item row="5" column="3" colspan="2">
     <widget class="QPushButton" name="sendButton">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(200, 200, 200);
background-color: rgb(33, 33, 33);
font: 8pt &quot;Century Gothic&quot;;</string>
      </property>
      <property name="text">
       <string>Отправить</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
